### DISCUSSION TITLE
請問有無RSS訂閱功能

### DISCUSSION THREAD

**Discussion by huawiz** [sdwh/sdwh.github.io#31] [Q&A]
版主你好，因為考AI900看到你的部落格，覺得不錯想訂閱，但是用RSS工具卻抓不到你部落格的RSS~

**Comment by sdwh** [comment]
Hi Hi

做了一點調整，試試看這個位址：https://sdwh.dev/atom.xml 😀

**Comment by huawiz** [comment]
可以用了~感謝
